scripts/zowe-install-zlux.sh

```diff
@@ -55,9 +63,11 @@ cp -f ${INSTALL_DIR}/files/zlux/config/plugins/* zlux-app-server/defaults/plugin
 
 echo "Unpax zssServer " >> $LOG_FILE
 cd zlux-app-server
-pax -r -px -f $INSTALL_DIR/files/zss.pax bin
+pax -r -px -f $INSTALL_DIR/files/zss.pax ${ZSS_COMPONENT_DIR}
 cd bin
-extattr +p zssServer
+# extattr +p zssServer
```

Comment:
comments not needed anymore, but, chmod +x may or may not be needed for the contents of the bin folder.
also, you may need to chmod 750 of app-server/share/zlux-app-server/defaults/serverConfig
and chmod 740 of app-server/share/zlux-app-server/defaults/serverConfig/*